build.sh

```diff
@@ -0,0 +1,121 @@
+
+#!/bin/bash
+
+function usage {
+    echo -e "Usage:"
+    echo -e "$0 CMD"
+    echo -e "Available CMDs:"
+    echo -e "\tvp2420 - build Protectli VP2410 coreboot image"
+    exit 1
+}
+
+if [ $# -le 0 ]; then
+    usage
+fi
+
+function buildImage {
+
+	if [ ! -d 3rdparty/blobs/mainboard ]; then
+		git submodule update --init --checkout
+	fi
+
+	if [ ! -d 3rdparty/blobs/mainboard/protectli ]; then
+		wget https://cloud.3mdeb.com/index.php/s/FzF5fjqieEyQX4e/download -O protectli_blobs.zip
+		unzip protectli_blobs.zip -d 3rdparty/blobs/mainboard
+		rm protectli_blobs.zip
+	fi
+
+	docker run --rm -it -v $PWD:/home/coreboot/coreboot \
+		-v $HOME/.ssh:/home/coreboot/.ssh \
+		-w /home/coreboot/coreboot coreboot/coreboot-sdk:b0d87f753c \
```

Comment:
```suggestion
		-w /home/coreboot/coreboot coreboot/coreboot-sdk:2021-09-23_b0d87f753c\
```

Was that even working?